tests/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "tests",
```

Comment:
Update the name to be more suitable
`Drupal Cypress tests`

Reply:
renames to 'cypress-drupal-tests'